<launch>
  <!-- 0-raw, 1-pos, 2-trj -->
  <param name="flight_mode" value="0" type="int" />

  <param name="vehicle_num" value="1" type="int" />
  <param name="joy_num" value="1" type="int" />
  <param name="radio_num" value="1" type="int" />

  <arg name="joy_dev0" default="/dev/input/js0" />

  <param name="radio_dev0" value="/dev/ttyUSB0" />
  <!--arg name="radio_dev1" value="/dev/ttyUSB1" /-->

  <!--include file="$(find vicon_bridge)/launch/vicon.launch"/-->





  <group ns="vehicle0">
    <param name="radio_index" value="0" type="int"/>
    <param name="index_in_radio" value="0" type="int"/>
    <param name="vehicle_addr_l" value="0xBE6D4E41" /> 
    <node pkg="licht_controls" name="controller0" type="Lichtcontroller" >
      <param name="ctrl_node_num" value="0" type="int"/>
      <rosparam 
        file="$(find licht_controls)/config/Lichtparam.yaml"
        command="load" 
      />
    </node>
  </group>


  <node pkg="licht_controls" name="Lichtlinker" type="Lichtlinker" output="screen">
    <param name="radio0_addr_l" value="0x616D4E41" /> 
  </node>
  <node pkg="licht_coordination" name="Lichtcoord_manuel" type="Lichtcoord_manuel" output="screen"/>

</launch>
